src/SearchInput/SearchInput.tsx

```diff
@@ -11,19 +12,29 @@ export type SearchInputItem = {
 
 interface IContainer {
```

Comment:
This one doesn't look like it's used for anything so can remove here and in the styled component `Container`